backend/utils/constants.py

```diff
@@ -1,3 +1,14 @@
+import os
+
+from django.conf import settings
+from utils.common_utils import CommonUtils
+
+os.environ.setdefault(
+    "DJANGO_SETTINGS_MODULE",
+    os.environ.get("DJANGO_SETTINGS_MODULE", "backend.settings.dev"),
+)
```

Comment:
@muhammad-ali-e why is this required? Ideally django would throw an error if this is not set I think - why do we require to set this here?

Reply:
The log_events function relies on constants from constants.py. Since this script runs outside the Django context, we need to manually initialize the Django settings module to ensure the necessary configuration is available like we had in celery.py, wsgi.py etc..